README.md

```diff
@@ -124,6 +124,7 @@ All icons used are the 16px variant available, which fit perfectly in normal tex
 | [`UnwatchRepository.svg`][OcticonsUnwatchRepository] | ![OcticonsUnwatchRepository] |       |
 | [`WatchRepository.svg`][OcticonsWatchRepository]     | ![OcticonsWatchRepository]   |       |
 | [`Wiki.svg`][OcticonsWiki]                           | ![OcticonsWiki]              |       |
+| [`People.svg`][OcticonsPeople]                       | ![OcticonsPeople]            |       |
```

Comment:
Also move it under `ForkedRepository.svg` please

Reply:
Hey @Andre601!
Head of docs, even I didn't understand the first time you said, he maybe not understood, you should have given detailed explanation the second time, something like: "cut the people row and add it under forked repo row"

Reply:
Done @PuneetGopinath